testnet/stacks-node/src/nakamoto_node/relayer.rs

```diff
@@ -748,7 +809,10 @@ impl RelayerThread {
         )?;
 
         let new_miner_handle = std::thread::Builder::new()
-            .name(format!("miner.{parent_tenure_start}"))
+            .name(format!(
+                "miner.{parent_tenure_start} (bound ({},{})",
+                &self.config.node.p2p_bind, &self.config.node.rpc_bind
```

Comment:
Can this use the port numbers instead of the full bind address? I can't think of a situation where the bind address is differentiating anything more than the bind ports already are, and it substantially reduces the width of the logs.

Reply:
Sure; I wasn't even intending to keep this new format anyway